### ISSUE TITLE
revisit maven patch after jflex update

### ISSUE THREAD

**Issue by tarzanek** [oracle/opengrok#2574]
latest RCs do this:
```
14:29:46 WARNING: ERROR addFile(): /var/opengrok/src/opengrok/testdata/sources/javascript/testlong.js
java.lang.IllegalArgumentException: Document contains at least one immense term in field="full" (whose UTF8 encoding is longer than the max length 32766), all of which were skipped.  Please correct the analyzer to not produce such terms.  The prefix of the first immense term is: '[101, 121, 106, 50, 122, 120, 106, 122, 97, 119, 57, 117, 105, 106, 111, 122, 108, 99, 106, 122, 98, 51, 118, 121, 121, 50, 118, 122, 105, 106]...', original message: bytes can be at most 32766 in length; got 180683
```
it seems the patch in maven for jflex doesn't kick in ... also while at it, it makes sense to enable this patch for every analyser !

**Comment by tarzanek** [comment]
and tests need fixing, since they continue and don't detect this (must be some silly check, since they are in place and test the positive outcome)

**Comment by tarzanek** [comment]
Vladimir Kotal [4:09 PM]
I noticed this too inthe logs
The log message should be fixed too - is it WARNING or ERROR ? :)

tarzanek [4:10 PM]
well warning, since indexing will continue, but the rest of the file after such token is ignored
(hence whole file with such token might be ignored)
I am quite this must be just small typo when the 1.7 jflex update happened
I will diff the generated files in 1.7 and 1.6 and adjust the patch
+ set it up for EVERY analyser

Vladimir Kotal [4:11 PM]
Yep, however the log string contains both levels

tarzanek [4:11 PM]
ok, will fix too

**Comment by tarzanek** [comment]
ok, so patch in mvn is OK, it is just not getting applied!

**Comment by vladak** [comment]
fixed in 3e49081e7c3321b4135887a9ecb233a60e99a8e4
